WowPacketParserModule.V9_0_1_36216/Parsers/AchievementHandler.cs

```diff
@@ -0,0 +1,31 @@
+﻿using WowPacketParser.Enums;
+using WowPacketParser.Misc;
+using WowPacketParser.DBC;
+using System.Collections.Generic;
+
+namespace WowPacketParserModule.V9_0_1_36216.Parsers
+{
+    public static class AchievementHandler
+    {
+        public static Dictionary<int, ulong> FactionReputationStore { get; } = new Dictionary<int, ulong>();
+
+        public static void ReadCriteriaProgress(Packet packet, params object[] indexes)
+        {
+            int criteriaId = packet.ReadInt32<CriteriaId>("CriteriaID", indexes);
+            ulong quantity = (ulong)packet.ReadInt64("Quantity", indexes);
```

Comment:
uint64

Reply:
Was copied from struct in WPP (will change)